api/v1alpha1/clusterinstance_types.go

```diff
@@ -166,6 +179,10 @@ type NodeSpec struct {
 	// +optional
 	SuppressedManifests []string `json:"suppressedManifests,omitempty"`
 
+	// PruneManifests represents a list of Kubernetes resource references that indicates which "node-level" manifests
+	// should be pruned (removed).
+	PruneManifests []ResourceRef `json:"pruneManifests,omitempty"`
```

Comment:
Does this need to be explicitly marked as optional?

Reply:
Good catch! Updated to explicitly mark as `optional` (although the `omitempty` json tag should be sufficient to flag the field as optional).